src/AlertBox/AlertBox.tsx

```diff
@@ -62,9 +62,9 @@ const TYPES = {
 };
 
 AlertBox.propTypes = {
```

Comment:
btw, why not include the modifications to apps dashboard react typings to the `minimal-react-typings` repo?